src/tools/pub/pubspec.md

```diff
@@ -380,7 +380,9 @@ the pattern is considered relative to the package's root directory.
 ### Screenshots
 
 Packages can showcase their widgets or other visual elements using screenshots
-displayed on their page.
+displayed on their page. The `screenshots` section is only intended for
+screenshots of the package in use. Do not use it for branding purposes such as
+logos.
```

Comment:
**issue:** Use active voice and condition, then instruction format, both per Google Style.
```suggestion
displayed on their page. To add screenshots for the package in use,
use the `screenshots` section. Don't use logos or other branding imagery
in this section.
```